tests/test_scoring.py

```diff
@@ -182,3 +236,34 @@ def test_nnscore():
             print(np.vstack((descs[mask, i], descs_correct[mask, i])))
 
     assert_array_almost_equal(descs, descs_correct, decimal=4)
+
+
+def test_model_train():
+    mols = list(oddt.toolkit.readfile('sdf', actives_sdf))[:10]
+    list(map(lambda x: x.addh(), mols))
+
+    rec = next(oddt.toolkit.readfile('pdb', receptor_pdb))
+    rec.protein = True
+    rec.addh()
+
+    data_dir = os.path.join(test_data_dir, 'data')
+    home_dir = mkdtemp()
+    pdbbind_versions = (2007, 2013, 2016)
+
+    for pdbbind_v in pdbbind_versions:
+        os.symlink(os.path.join(data_dir, 'pdbbind'),
```

Comment:
Creating symlinks will fail if they exist. I'd check for existance here and remove it afterwards if the file is symlink.